vespamalloc/src/vespamalloc/malloc/datasegment.h

```diff
@@ -8,19 +8,36 @@
 
 namespace vespamalloc::segment {
 
-template<typename MemBlockPtrT>
+class IMemblockInfo {
```

Comment:
Nit: should this be `IMemBlockInfo` to match the previous `MemBlockPtrT` casing?

Reply:
Fixed